docs/services/data/_index.md

```diff
@@ -204,6 +272,9 @@ The following attributes are available for the data management service:
 | `sync_interval_mins` | float | Optional | Time interval in minutes between syncing to the cloud. Viam does not impose a minimum or maximum on the frequency of data syncing. However, in practice, your hardware or network speed may impose limits on the frequency of data syncing. <br> Default: `0.1`, meaning once every 6 seconds. |  <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
 | `delete_every_nth_when_disk_full` | int | Optional | How many files to delete when local storage meets the [fullness criteria](/services/data/#storage). The data management service will delete every Nth file that has been captured upon reaching this threshold. Use JSON mode to configure this attribute. <br> Default: `5`, meaning that every fifth captured file will be deleted. |  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
 | `maximum_num_sync_threads` | int | Optional | Max number of CPU threads to use for syncing data to the Viam Cloud. <br> Default: [runtime.NumCPU](https://pkg.go.dev/runtime#NumCPU)/2 so half the number of logical CPUs available to viam-server |  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
+| `mongo_capture_config.uri` | string | Optional | The [MongoDB URI](https://www.mongodb.com/docs/v6.2/reference/connection-string/) data capture will attempt to write tabular data to after it is enqueued to be written to disk. When non empty, data capture will capture tabular data to the configured MongoDB database & collection at that URI.<br>See  `mongo_capture_config.database` and  `mongo_capture_config.collection` below for database & collection defaults.<br>See [Data Capture Directly To MongoDB](/services/data/#capture-directly-to-mongodb) for an example config.|  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
```

Comment:
nit - 'data capture' is not a service from the user's perspective I think?

```suggestion
| `mongo_capture_config.uri` | string | Optional | The [MongoDB URI](https://www.mongodb.com/docs/v6.2/reference/connection-string/) that the data management service will attempt to write tabular data to after it is enqueued to be written to disk. When non empty, the data management service will capture tabular data to the configured MongoDB database & collection at that URI.<br>See  `mongo_capture_config.database` and  `mongo_capture_config.collection` below for database & collection defaults.<br>See [Data Capture Directly To MongoDB](/services/data/#capture-directly-to-mongodb) for an example config.|  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
```

Reply:
Data Capture is not a service but it is a feature of Data Manager. I think this is indeed in line with the existing nomenclature of these docs.

See at the top of the docs:
![Screenshot 2024-11-14 at 10 09 10 AM](https://github.com/user-attachments/assets/32047db9-cfe2-4c86-9618-df8f6109590a)

I think it is important to document that this is a feature of Data Capture as users need to understand that if they disable data capture they will stop writing data to mongo, where as if they are capturing & not syncing, they will still write data to mongo.

Reply:
ok -- I was worried of referring to Data Capture (capital D capital C) for the first time in the doc. in your screenshot, the word 'capture' is used for its functional meaning, not as a name of a _thing_. I'll yield to your judgement and have approved the change already, just wanted to note this difference.

Reply:
I'll downcase data capture to keep it consistent with the rest of the docs.

Reply:
oh yeah that works -- thanks! :)